requirements.txt

```diff
@@ -5,4 +5,5 @@ git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.nltk&subdirectory=
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.elastic&subdirectory=src/elastic
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.allennlp&subdirectory=src/allennlp
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.spacy&subdirectory=src/spacy
+git+https://git@github.com/asyml/forte-wrappers#egg=forte-wrappers[scispacy]
```

Comment:
We will probably not do it this way.

Could you help add `scispacy` as an extra requirement for spacy here: https://github.com/asyml/forte-wrappers/blob/main/src/spacy/setup.py

Reply:
This file should be deleted (I forgot): https://github.com/asyml/forte-wrappers/blob/main/setup.py

Reply:
I tried to add scispacy in spacy setup.py and delete setup.py in main: https://github.com/asyml/forte-wrappers/tree/twitter_rename, but the pipeline failed at mypy check. I will try to find out a way to make it back to work.

Reply:
I think we just run mypy multiple times?  Like `mypy src/allennlp`, you can loop over the directories: https://github.com/asyml/forte-wrappers/blob/main/.github/workflows/main.yml#L156